File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.cpp
class Solution {
public:
    int reverseDegree(string s) {
        int n= s.length();
        int sum=0;
        for(int i=1;i<=n;i++){
            sum+=(26-(s[i-1]-'a'))*i;
            
        }
        return sum;
    }
};

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.js
/**
 * @param {string} s
 * @return {number}
 */
var reverseDegree = function (s) {
    let ans = 0;
    for (let i = 1; i <= s.length; i++) {
        ans += (26 - (s.charCodeAt(i - 1) - 'a'.charCodeAt(0))) * i;
    }
    return ans;
};